aya/src/sys/pid_fd.rs

```diff
@@ -0,0 +1,31 @@
+use crate::sys::{SysResult, Syscall};
+use libc::pid_t;
+use std::os::fd::{AsRawFd, RawFd};
+/// A file descriptor of a process.
+pub(crate) struct PidFd(RawFd);
```

Comment:
```suggestion
pub(crate) struct PidFd(MockableFd);
```

or `OwnedFd`. We shouldn't be dealing with `RawFd` directly.

Reply:
Done.
<!-- Reviewable comment -OIKI2j16_6TWrnEkKgG:b-896fix -->
<!-- Sent from Reviewable.io -->